packages/itwinui-css/src/menu/menu.scss

```diff
@@ -65,6 +71,13 @@ $iui-active-outline: thin solid var(--iui-color-border-accent);
       margin-right: 0;
       margin-left: var(--iui-size-xs);
     }
+
+    &.iui-multilevel-selected {
+      display: flex;
+      justify-content: flex-end;
+      flex-shrink: 0;
```

Comment:
Adding `flex-shrink: 0` does not work too well when the `.iui-multilevel-selected` content is too long.

![image](https://user-images.githubusercontent.com/45748283/215620424-f0dbdc49-c4d1-47d4-a2f9-881884159efa.png)

Removing this gave the below result:

![image](https://user-images.githubusercontent.com/45748283/215620429-dbd23360-cf3b-43e4-a097-d751cd7c2c89.png)

Also, should there be a `min-width` (e.g. 30-70 split) between left and right?

Reply:
Hmm, min-width % seems like a good way to fix it. I added `flex-shrink: 0` because of 
![image](https://user-images.githubusercontent.com/37936169/215803835-abd6ac49-c5b2-47d4-9272-7f80d2358491.png)

Reply:
I didn't use `min-width`, but I did fix this issue by changing the css used for `iui-multilevel-selected`. I also added an example for this scenario. I did a 30/title - 70/selection split, but I can change that to something else if need be.